### PR TITLE
Hotkeys: Use Keyboard Layout

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: luaui/configs/keyboard_layouts.lua [hunk 1]
```diff
@@ -19,28 +30,41 @@ local keyLayouts = {
 			[4] = "V",
 			[5] = "B",
 			[6] = "N",
+			[7] = "M",
 		}
 	},
 	dvorak = {
 		[3] = {
```
Comment (rafidka) [inline luaui/configs/keyboard_layouts.lua:37]: Why are the rows incomplete?

Comment (badosu) [reply luaui/configs/keyboard_layouts.lua:37]: Only keys that are used need to be defined

Comment (rafidka) [reply luaui/configs/keyboard_layouts.lua:37]: `[` and `]` are used, no? Also, what if someone defines his/her own keys that are not part of this map?

Comment (badosu) [reply luaui/configs/keyboard_layouts.lua:37]: The widget concerns only to what it binds and unbinds, it does not fix external keys

Comment (badosu) [reply luaui/configs/keyboard_layouts.lua:37]: We could define what [ and ] is positionally but even in qwerty keyboards those keys are not consistently placed

Comment (Beherith) [reply luaui/configs/keyboard_layouts.lua:37]: Yeah [ ] is totally regional :/

**Pull request merged**